### DISCUSSION TITLE
TIFF -> PNG conversion distorted

### DISCUSSION THREAD

**Discussion by bogaret** [ImageMagick/ImageMagick#5617] [Help] [answered]
Command ran: `convert file_example_TIFF_1MB.tiff file_example_TIFF_1MB.png`

![file_example_TIFF_1MB](https://user-images.githubusercontent.com/114960875/193700595-18df7f02-7477-45d9-90cd-1693cf7a5676.png)

Seems to happen with all tiff files I'm trying the command with. Version is 7.1.0-47. Any idea what I'm doing and why the image is repeating?

Output: 
[files.zip](https://github.com/ImageMagick/ImageMagick/files/9701559/files.zip)

**Comment by snibgo** [comment, marked as answer]
V7.1.0-47 had problems with TIFF. I suggest you upgrade to 7.1.0-49 or later and try again.

**Reply by bogaret** [reply]
This seems to have solved the issue, thank you!

**Comment by fmw42** [comment]
The command

`convert file_example_TIFF_1MB`

is not valid. You have no output image. Imagemagick does not automatically overwrite the input with the output unless the same name is used for the output.

**Reply by bogaret** [reply]
Sorry mistyped, but it seems like the upgrade to 7.1.0-49 fixed the issue.
